lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystemE2E.cs

```diff
@@ -49,15 +50,16 @@ public TestAzureBlockBlobFileSystemE2E()
                 .Build();
 
             _fileSystem = TangFactory.GetTang().NewInjector(conf).GetInstance<AzureBlockBlobFileSystem>();
-            _container = CloudStorageAccount.Parse(ConnectionString).CreateCloudBlobClient().GetContainerReference(defaultContainerName);
-            _container.CreateIfNotExistsAsync().Wait();
+            _client = CloudStorageAccount.Parse(ConnectionString).CreateCloudBlobClient();
+            _container = _client.GetContainerReference(defaultContainerName);
+            _container.CreateIfNotExists();
         }
 
         public void Dispose()
         {
             if (_container != null)
             {
-                _container.DeleteIfExistsAsync().Wait();
+                _container.DeleteIfExists();
```

Comment:
Hi @dwaijam , 

The Azure Storage Library is not consistent with their api -- they have a DeleteIfExists on .net45 libraries, but they have removed this call for .net core. Since we want to support .net core, I have converted these apis to use the Async call with the Wait. So please keep these in place so we dont have to change them back later. 

Also this comment applies to all DeleteIfExists cases. Thank you!

Reply:
Done